core/templates/dev/head/components/SharingLinksDirective.js

```diff
@@ -45,6 +52,42 @@ oppia.directive('sharingLinks', [function() {
 
         $scope.escapedTwitterText = (
           oppiaHtmlEscaper.unescapedStrToEscapedStr($scope.getTwitterText()));
+
+        $scope.isNotDashboard = function() {
```

Comment:
I think all this code can also disappear? Don't forget to merge from the latest version of upstream/develop into your branch, too, in order to get the latest version of this directive.